metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -186,6 +186,11 @@
 except ImportError:
     REAUTHENTICATE_ERRORS = (NonXMLResponseError,)
 
+RETRIABLE_ERROR_CODES = {
```

Comment:
open questions:

* should we include others such as `503 service Unavailable`, `408 Request Timeout`, ....?
* should we make this configurable in the recipe?

Reply:
> should we include others such as 503 service Unavailable, 408 Request Timeout, ....?

We can add [these](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py#L290) codes for retry.  Can add 408 if we got that from tableau and if request is likely to succeed on retry although given it represents client side idleness, that may not be relevant here.


> should we make this configurable in the recipe?

I believe - making it configurable is okay for one-off debugging but would not help us make source more robust, as we would ideally always want to update source such that known retriable error codes are always retried. So I would rather not add it.

Reply:
I have aligned on existing retrievable error codes and added 408

I didn't know about the retry configuration at Tableau `Server` level. Either it is not working as expected or not correctly configured.

Reply:
True, this needs to be looked into.